### DISCUSSION TITLE
autohiding navbar shows too soon when tabbar is hidden

### DISCUSSION THREAD

**Discussion by MoscaDotTo** [black7375/Firefox-UI-Fix#994] [General]
Hello! I fixed this for myself at one point but accidentally overwrote my change so I'm asking here! 

I want the navbar to only unhide itself when my cursor touches the topmost row of pixels on the screen. currently, it unhides itself whenever the mouse is about 30px from the top of the screen (anywhere within where the navbar *would* be if it were visible) which actively interferes with interactive website elements. can someone point me to the relevant line of css to modify?

**Comment by segln** [comment]
Have you resolved it? I would really appreciate it if you could share the solution.
